.rubocop.yml

```diff
@@ -214,6 +214,17 @@ Style/PercentLiteralDelimiters:
     '%q': '{}' # Chosen for module descriptions as () are frequently used characters, whilst {} are rarely used
   VersionChanged: '0.48.1'
 
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Description: 'This cop enforces the presence (default) in method calls containing parameters.'
+  IgnoredPatterns:
+    - ^require
+    - ^raise
+    # Gemfile DSL
+    - ^gemspec
+    - ^gem
+    - ^source
```

Comment:
I've opted out of the common DSL patterns; I've ran rubocop -a on our Gemfile, there's no changes now other than a space after a comment

Reply:
I can add "puts" to this list too if anyone has strong opinions. In terms of workflow, write whatever code comes to your fingers - then automate the linting process with `rubocop -a` to give us a consistent codebase.